Sources/Plasma/PubUtilLib/plModifier/plImageLibMod.h

```diff
@@ -75,6 +75,9 @@ class plImageLibMod : public plSingleModifier
     };
 
     size_t  GetNumImages() const { return fImages.size(); }
+    plBitmap* GetImage(const ST::string&) const;
+    const std::vector<plBitmap*> GetImages() const;
+    const std::vector<ST::string> GetImageNames() const;
```

Comment:
```suggestion
    std::vector<ST::string> GetImageNames() const;
```

You're giving the user a new container - it's ok if they modify it, I think.